src/adapters/givPowerBalanceAggregator/givPowerBalanceAggregatorInterface.ts

```diff
@@ -0,0 +1,48 @@
+export interface GetLeastIndexedBlockTimeStampInputParams {
+  networks?: string; // comma separated sample: 100,420
+  network?: string | number; // comma separated sample: 100
+}
+
+export interface GetLatestBalanceOfAnAddressInputParams
+  extends GetLeastIndexedBlockTimeStampInputParams {
+  address: string; // Ethereum wallet address
+}
+
+export interface GetBalanceOfAddressesInputParams
+  extends GetLeastIndexedBlockTimeStampInputParams {
+  timestamp: number; // sample: 1691739112
+  addresses: string; // comma separated sample: walletAddress1,walletAddress2,..
```

Comment:
Let's transfer it to appropriate format in the adapter
```suggestion
  addresses: string[]; 
```

Reply:
Done